docs/claimantmodel/README.md

```diff
@@ -0,0 +1,113 @@
+# Sigstore Claimant Model
+
+Sigstore's claimant model includes a set of claims produced by Rekor and Fulcio logs. See [Claimant Model](https://github.com/google/trillian/blob/master/docs/claimantmodel/CoreModel.md) for more information and terminology.
+
+Sequence diagrams generated with [Claimant Model Render Tool](https://github.com/google/trillian/tree/master/docs/claimantmodel/experimental/cmd/render). This tool is a work in progress, so there may be some errors in the generated models.
+
+Each claim is in `model.md` in each folder. `full.md` contains the claim made in both the domains of Rekor or Fulcio and the log. The sequence diagram shows the interactions between all Actors. 
+
+All claims are also provided in this readme.
+
+## Fulcio: Identity-based signing
+
+<dl>
+<dt>Claim<sup>Fulcio</sup></dt>
+<dd><i>${OIDCIDOwner} authorizes ${CA} to bind ${PubKey} to ${OIDCIdentity}</i></dd>
+<dt>Statement<sup>Fulcio</sup></dt>
+<dd>X.509 certificate containing ${PubKey} and ${OIDCIdentity}, signed by ${CA}</dd>
+<dt>Claimant<sup>Fulcio</sup></dt>
+<dd>${CA}</dd>
+<dt>Believer<sup>Fulcio</sup></dt>
+<dd>Software Installer</dd>
+<dt>Verifier<sup>Fulcio</sup></dt>
+<dd>${OIDCIDOwner}: <i>${OIDCIDOwner} authorizes ${CA} to bind ${PubKey} to ${OIDCIdentity}</i></dd>
+<dt>Arbiter<sup>Fulcio</sup></dt>
+<dd>Community</dd>
+</dl>
+
+## Rekor: Identity-based signature
+
+<dl>
+<dt>Claim<sup>Rekor</sup></dt>
+<dd><i>${OIDCIdentity} signs ${Hash} using the key bound by ${Certificate}</i></dd>
+<dt>Statement<sup>Rekor</sup></dt>
+<dd>${Hash}, X.509 certificate ${Certificate} containing ${OIDCIdentity}, and signature over ${Hash}</dd>
+<dt>Claimant<sup>Rekor</sup></dt>
+<dd>${OIDCIdentity}</dd>
```

Comment:
After reviewing the following CM I recommend OIDCIDOwner

Reply:
The reason we had went with OIDCIdentity/Key was because I could find a valid signature lying around and upload it to the log. It likely wouldn't be usable in that case, but the claim is not guaranteed to be produced by the owner.

Do you think that distinction is too subtle and we should focus on the expected use case?

Reply:
The Claim is that $someone signs $hash using a key bound in $cert. This is orthogonal to uploading it a log.

The only entity that can sign that hash is the owner of the private key material, which _should_ be the `OIDCIDOwner`, right?

Reply:
That's correct. I've updated.